docs/source/algorithms/LoadMergeRuns-v1.rst

```diff
@@ -0,0 +1,62 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm loads multiple runs conforming the rules in :py:obj:`MultipleFileProperty <mantid.api.MultipleFileProperty>`.
+When listing is requested, the output is a :ref:`WorkspaceGroup <WorkspaceGroup>`, containing items for each run.
+When summing is requested, it is performed pair-wise *in-situ* with :ref:`MergeRuns <algm-MergeRuns>`.
+By doing this, the metadata (SampleLogs) are also merged governed by the rules specified in the Instrument Parameter File (IPF)
+as explained in :ref:`MergeRuns <algm-MergeRuns>`.
+This rules can be overridden by the **MergeRunsOptions** input.
+Specific loader can be given by the **LoaderName** and **LoaderVersion**.
+If left as defaults, the algorithm will automatically find out the corresponding specific loader, deducting from the first run in the list.
+It will set the name and the version of the suitable loader found as output properties in **LoaderName** and **LoaderVersion**.
```

Comment:
is "deducting" the correct verb?

Reply:
Oops I intended to comment the previous line.

Reply:
Nope, I meant deduce